scripts/install-windows-prereqs.ps1

```diff
@@ -360,6 +362,31 @@ function Install-Python3 {
     Move-Item -Path "$tempInstallDir\*" -Destination $installDir
     Add-ToSystemPath -Path $installDir
 
+    Start-ExecuteWithRetry -ScriptBlock {
+        # Install the PIP installer
+        $Destination = Join-Path $installDir "get-pip.py"
+        Start-FileDownload -URL $GetPipURL `
+                           -Destination $Destination
+        $downloaded_hash = Get-FileHash $Destination
+        if ($downloaded_hash.Hash -ne $GetPipHash)
+        {
+            Throw "Error: Computed hash ($downloaded_hash) does not match expected hash ($GetPipHash)"
+        }
+        else
+        {
+            Write-Output "Computed hash ($downloaded_hash) matches expected hash ($GetPipHash)"
+        }
+
+        # Install PIP
+        python $Destination
+        $Scripts = Join-Path $installDir "Scripts"
+        $env:PATH += ";" + $Scripts
```

Comment:
This only updates the `PATH` for this instance of the shell; it's sufficient for allowing the next script block to succeed, Shouldn't this just call `Add-ToSystemPath` instead, which updates the current shell $env:PATH as well as the global `PATH`? (The existing function also doesn't append duplicates)

Reply:
Done